src/components/SignUp/SignUp.js

```diff
@@ -0,0 +1,149 @@
+          />
+          <p className="text-red-800 font-semibold">
+            {' '}
+            {errors?.lastName?.message}{' '}
+          </p>
+        </label>
+        <label htmlFor="email" className="mt-3">
+          <span className="text-primary font-semibold">Email</span>
+          <input
+            type="email"
+            name="email"
+            placeholder="Email"
+            {...register('email')}
+            className="sm:w-96 w-80 shadow-lg text-primary focus:outline-none focus:tertiary focus:ring-1 focus:ring-tertiary rounded-md placeholder:italic placeholder:text-tertiary px-3 py-1 mt-1 block duration-500"
+          />
+          <p className="text-red-800 font-semibold">
+            {' '}
+            {errors?.email?.message}{' '}
+          </p>
+        </label>
+        <label htmlFor="password" className="mt-3">
+          <span className="text-primary font-semibold">Password</span>
+          <input
+            type="password"
+            name="password"
+            placeholder="Password"
+            {...register('password')}
+            className="sm:w-96 w-80 shadow-lg focus:outline-none focus:tertiary focus:ring-1 focus:ring-tertiary text-primary rounded-md placeholder:italic placeholder:text-tertiary px-3 py-1 mt-1 block duration-500"
+          />
+          <p className="text-red-800 font-semibold">
+            {' '}
+            {errors?.password?.message}{' '}
+          </p>
+        </label>
+        <label htmlFor="confirmPassword" className="mt-3">
+          <span className="text-primary font-semibold">Confirm Password</span>
+          <input
+            type="password"
+            name="confirmPassword"
+            placeholder="Confirm Password"
+            {...register('confirmPassword')}
+            className="sm:w-96 w-80 shadow-lg focus:outline-none focus:tertiary focus:ring-1 focus:ring-tertiary text-primary rounded-md placeholder:italic placeholder:text-tertiary px-3 py-1 mt-1 block duration-500"
+          />
+          <p className="text-red-800 font-semibold mt-1 mb-3">
+            {' '}
+            {errors?.confirmPassword?.message && 'Passwords should match'}{' '}
+          </p>
+        </label>
+        <button
+          type="submit"
+          className="sm:w-96 w-80 font-semibold shadow-lg my-2 bg-primary text-background py-2 rounded-md hover:bg-tertiary hover:text-primary duration-500"
+        >
+          Sign Up
+        </button>
+        <p className="text-lg text-primary self-center mt-2">
+          Already Have an Account?{' '}
+          <Link
+            to="/signin"
+            className="underline hover:text-tertiary duration-300"
+          >
+            Sign In
+          </Link>
+        </p>
+        <p className="text-2xl text-primary font-semibold self-center my-4">
```

Comment:
the font size is too big in sign up with facebook and google

Reply:
I agree. Will fix that

Reply:
Agreed.